### PR TITLE
Prevent empty harvest run to fail

### LINKED ISSUE #1394: Harvester should not start a chord when there is no items to process
## Detailed Description
Some harvest jobs fail because there is no item to process.
Celery chords needs to have at least one item to process.

The following sentry error happen on such cases: https://sentry.data.gouv.fr/share/issue/9a3113bc1ab74f0ca0392a300f202b01/

## Possible Implementation
Prevent Celery from starting an empty chord when there is no item to process

## Your Environment
`udata==1.2.9`

### PR DESCRIPTION
**Pull request by noirbizarre**
Prevent empty harvest jobs to fail with Celery

Fix #1394

### CODE REVIEW

File: udata/harvest/tasks.py [hunk 1]
```diff
@@ -18,12 +18,15 @@ def harvest(self, ident):
     source = HarvestSource.get(ident)
     Backend = backends.get(source.backend)
     backend = Backend(source)
-    if backend.perform_initialization():
+    items = backend.perform_initialization()
+    if items > 0:
```
Comment (abulte) [inline udata/harvest/tasks.py]: `if items` and `else`?

Comment (noirbizarre) [reply udata/harvest/tasks.py]: No because failed bacth returns `None`

**Comment by noirbizarre** [comment]
https://sentry.data.gouv.fr/etalab/www-datagouvfr/issues/123361/

**Comment by noirbizarre** [comment]
https://sentry.data.gouv.fr/etalab/www-datagouvfr/issues/152712/

**Pull request merged**